BSPConvert.Lib/Source/MaterialConverter.cs

```diff
@@ -233,9 +238,23 @@ private void AppendShaderParameters(StringBuilder sb, Shader shader)
 				if (textureStage.rgbGen.HasFlag(ColorGen.CGEN_CONST))
 				{
 					var color = textureStage.constantColor;
-					var colorStr = $"{color[0]} {color[1]} {color[2]}";
-					sb.AppendLine("\t$color \"{" + colorStr + "}\"");
+					var r = Math.Round(255 * Math.Pow((float)color[0] / 255, 2.2), 2); // gamma correct rgb values
+					var g = Math.Round(255 * Math.Pow((float)color[1] / 255, 2.2), 2);
+					var b = Math.Round(255 * Math.Pow((float)color[2] / 255, 2.2), 2);
+
+					if (lightmapped)
```

Comment:
It's preferable to write the code like this:
```
string colorStr;
if (lightmapped)
    colorStr = $"{Math.Round(r / smoothShadows, 2)} {Math.Round(g / smoothShadows, 2)} {Math.Round(b / smoothShadows, 2)}"; // apply custom brightness settings on lightmapped textures
else
    var colorStr = $"{r} {g} {b}";

sb.AppendLine("\t$color2 \"{" + colorStr + "}\"");
```

That way, the `AppendLine` code isn't duplicated.